dataland-frontend/tests/component/components/resources/frameworkDataSearch/euTaxoNonFinancials/EuTaxonomyForNonFinancialsPanel.cy.ts

```diff
@@ -80,4 +83,74 @@ describe("Component test for the NewEUTaxonomy Page", () => {
         .should("contain", gammaContributionToClimateChangeMitigation);
     });
   });
+  it("Checks if the reports banner and the corresponding modal is properly displayed", () => {
+    const reportsAndReportingPeriods =
+      extractReportsAndReportingPeriodsFromDataAndMetaInfoSets(mockedBackendDataForTest);
+    const hightestIndexOfReportingPeriods = calculateIndexOfNewestReportingPeriod(reportsAndReportingPeriods[1]);
+    cy.mountWithDialog(
+      ShowMultipleReportsBanner,
+      {
+        keycloak: minimalKeycloakMock({}),
+      },
+      {
+        reports: reportsAndReportingPeriods[0],
+        reportingPeriods: reportsAndReportingPeriods[1],
+      },
+    ).then(() => {
+      cy.get(`[data-test="frameworkNewDataTableTitle"`).contains(
+        "Data extracted from the company report.Company Reports",
```

Comment:
Space after `report. ` .. Should be ` report. Company `

Reply:
done